src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -655,6 +656,7 @@ This resets the cluster to its initial state.`,
 			fmt.Println("CTRL-C to exit")
 			ch := make(chan os.Signal, 1)
 			signal.Notify(ch, os.Interrupt)
+			signal.Notify(ch, syscall.SIGHUP)
```

Comment:
looks like notify actually takes `... os.Signal` so you could combine these. I don't know a lot about signals, does it make sense to include the other termination signals here?

Reply:
good call on making it all one line.  will do that.

Yeah, I think we could add SIGTERM here.  Will do that.  Everything else (like SIGQUIT / Control-\) I think we should just let be unhandled; if someone is manually sending signals here then maybe they have some reason to exit uncleanly.  SIGTERM is sent if you are shutting down your computer, I think, so that's a reasonable time to clean up.